tests/register/register.go

```diff
@@ -107,9 +119,34 @@ func (test Test) UnmountPartitions(t *testing.T, loopDevice string) {
 	util.MustRun(t, "umount", fmt.Sprintf("%sp9", loopDevice))
 }
 
-func (test Test) RunCoreOSInstall(t *testing.T, loopDevice string, opts ...string) {
+func (test Test) GetInstallOptions(t *testing.T, loopDevice string, opts ...string) []string {
 	opts = append(opts, "-d", loopDevice)
 
+	// use local http server if we aren't testing specific flags
+	if test.Version == nil && test.BaseURL == nil && test.Channel == nil && test.Board == nil && !test.UseLocalFile {
```

Comment:
This makes it impossible to use a completely default command line (with only `-d` specified).  It seems as though we should have one test, at least, that does that.  Maybe add e.g. a `NoLocalServer` flag?